packages/carbon-component-mapper/src/files/plain-text.d.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactNode } from "react";
+
+export interface PlainTextProps {
+  label: ReactNode;
+  element?: string;
```

Comment:
There should be a list somewhere of acceptable element strings.

Reply:
Do you think it’s possible to generate this? Otherwise, it can be really anything, because Carbon is using global styles and does not have specific components for texts.

Reply:
Well, the plain text should be anything that uses text. So we are talking about headings, paragraphs, labels, etc. I think either PF4 or PF3 mappers have the list so we can make it a common props. Or maybe it is already in common props. By leaving this as a string, we are basically allowing it to be any element and we are getting to a `dangerouslySetInnerHTML` territory.

Reply:
I don't think it matters here. Programmers with the access to typescript can actually do everything they want to do, so I don't see why to limit them. 

And to prevent outside attacks we should do a real-time check in the render of the function, but we are not doing that in other mappers and maybe we should let application teams to handle that. (With a component wrapper or with a `resolveProps` function.)